### PR TITLE
Add/rename Linux SNMP templates for Zabbix 6.0+

### PR DESCRIPTION
**Pull request by teluq-pbrideau**
#### Pull Request (PR) description
 As mentionned in #851, the SNMP templates has been renamed upstream, this is an attempt to fix the tests

### CODE REVIEW

**Comment by teluq-pbrideau** [comment]
But I have no idea how to fix the archlinux tests:
```
Beaker::Host::CommandFailure:
  Host 'archlinuxrolling-64.example.com' exited with 1 running:
   puppet module install puppetlabs-postgresql -v 7.5.0
  Last 10 lines of output were:
  	Could not find 'english' (>= 0.a) among 97 total gem(s)
  	Checked in 'GEM_PATH=/root/.local/share/gem/ruby/3.0.0:/usr/lib/ruby/gems/3.0.0' , execute `gem env` for more information
```
Does someone know anything about this cryptic error?  Should we simply disable the archlinux tests suite?

**Comment by kenyon** [comment]
No need to address the archlinux test failure in this PR. That is happening on all of our modules and needs to be fixed in the files that are shared by all of the modules.

**Review by smortex — APPROVED** [review]

File: spec/acceptance/zabbix_host_spec.rb [hunk 1]
```diff
@@ -29,8 +29,12 @@
                         ['Template OS Linux SNMPv2']
                       when '5.0'
                         ['Template OS Linux SNMP']
-                      else
+                      when '5.2'
+                        ['Linux SNMP']
+                      when '5.4'
```
Comment (smortex) [inline spec/acceptance/zabbix_host_spec.rb:34]:
```suggestion
                      when '5.2', '5.4'
```

**Comment by smortex** [comment]
This looks good, I will merge it unless CI detect some kind of breakage.

I think it is much more of an enhancement for bringing support for newer versions of Zabbix than a bugfix, so maybe we can change the PR title to something like "Add Linux SNMP templates for Zabbix 6.0+" which will have move value when read in the changelog without context?

**Comment by teluq-pbrideau** [comment]
> I think it is much more of an enhancement for bringing support for newer versions of Zabbix than a bugfix, so maybe we can change the PR title to something like "Add Linux SNMP templates for Zabbix 6.0+" which will have move value when read in the changelog without context?

It was a change from upstream in 6.0.11, it was still named `Linux SNMP` in 6.0.10:
https://git.zabbix.com/projects/ZBX/repos/zabbix/browse/templates/os/linux_snmp_snmp/template_os_linux_snmp_snmp.yaml?at=refs%2Ftags%2F6.0.10

**Comment by teluq-pbrideau** [comment]
well... Is zabbix 6 still supported on centos7?
```
  Error downloading packages:
    zabbix-agent-6.0.12-release1.el7.x86_64: [Errno 256] No more mirrors to try.
  Error: /Stage[main]/Zabbix::Agent/Package[zabbix-agent]/ensure: change from '5.4.12-1.el7' to '0:6.0.12-release1.el7' failed: Could not update: Execution of '/usr/bin/yum -d 0 -e 0 -y update zabbix-agent' returned 1: Delta RPMs disabled because /usr/bin/applydeltarpm not installed.
```

**Comment by smortex** [comment]
> well... Is zabbix 6 still supported on centos7?

Packages seems to exist. Maybe a transient failure?
https://repo.zabbix.com/zabbix/6.0/rhel/7/x86_64/

**Comment by smortex** [comment]
Great, we will assume the failure was due to the repo being updated while the test suite was running :shrug:

**Comment by teluq-pbrideau** [comment]
Thanks @smortex for all the help to merge my stuff!

**Pull request merged**